package-deploy.yaml

```diff
@@ -1,10 +1,23 @@
 agents:
-  - name: deb
-    dockerFilePath: docker/build/cicd.ubuntu.Dockerfile
-    image: algorand/cicd-ubuntu
+  - name: docker
+    dockerFilePath: docker/build/docker.ubuntu.Dockerfile
```

Comment:
I think calling a dockerfile docker.ubuntu.Dockerfile might be redundant. I think cicd makes sense since these images have all the tools etc that can do our cicd process. Alternatively just calling them os.Dockerfile could work too

Reply:
I'm open to renaming it.  Basically, I need a dockerfile that is installing docker because it's needed for building the docker image.

I'm open to a better way of doing this.

Reply:
Thinking about this more, I think renaming these files will break our other mule yamls

Reply:
Let's talk about this briefly again tomorrow so I know what needs to be done.

Reply:
@bricerisingalgorand can you note where this will break in the other mule yamls so we can track this down? Also are these named docker because they're doing DIND?

Reply:
`dockerFilePath: docker/build/cicd.ubuntu.Dockerfile` changing this name would break the other mule yamls. Also these images are in dockerhub with the name `algorand/go-algorand-docker-linux-ubuntu`, so if we keep using that name running the tasks will be faster since the image won't have to be built on each computer

Reply:
I'm not changing the name, it's using a different dockerfile.

Reply:
Should this combine with the cicd docker image or does it need to be separate?

Reply:
It's separate because it needs to install docker to create the docker hub image.  It could use the cicd image and then install docker in the shell script, but I wanted to avoid doing it that way, at least for now.

This does need improvement, and I have some ideas for that, I'm hoping we can get this merged and then continue to improve upon it.

Reply:
ohhh ok I didn't realize it was a different dockerfile. Interesting, so this is for building the algod docker image right? I think this works for now, but maybe later on we can figure out how to use the mule runtime to execute the docker build since it has a daemon available to it